src/Umbraco.Tests.UnitTests/Umbraco.Core/Extensions/UriExtensionsTests.cs

```diff
@@ -47,12 +46,8 @@ public void Setup()
         [TestCase("http://www.domain.com/umbraco/test/legacyAjaxCalls.ashx?some=query&blah=js", "", true)]
         public void Is_Back_Office_Request(string input, string virtualPath, bool expected)
         {
-            
-            var mockHostingSettings = Mock.Get(_settingsForTests.GenerateMockHostingSettings());
-            mockHostingSettings.Setup(x => x.ApplicationVirtualPath).Returns(virtualPath);
-            var hostingEnvironment = new AspNetCoreHostingEnvironment(mockHostingSettings.Object, _hostEnvironment);
-
             var source = new Uri(input);
+            var hostingEnvironment = CreateHostingEnvironment(virtualPath);
```

Comment:
I know we use var as a rule (it might even be in a rule?) but I think the readability is improved with an explicit type - unless what **var** is, is obvious (e.g. string, bool, or a newed up object)

Reply:
IMO, the variable name is good here. I would have agreed if the variable name was just `h` or `he`. I'm pretty confident that I get an implementation of `IHostingEnvironment`.